src/main/java/edu/cornell/kfs/coa/businessobject/SubAccountGlobal.java

```diff
@@ -38,28 +62,130 @@ public SubAccountGlobal() {
 		return objectsToDeactivate;
 	}
 
+	/**
+	 * @see org.kuali.rice.krad.bo.GlobalBusinessObject#generateGlobalChangesToPersist()
+	 */
 	@Override
 	public List<PersistableBusinessObject> generateGlobalChangesToPersist() {
 		List<PersistableBusinessObject>  changesToPersist = new ArrayList<PersistableBusinessObject>();
-		
-		if(inactivate){
-			for(SubAccountGlobalDetail subAccountGlobalDetail : subAccountGlobalDetails){
-				subAccountGlobalDetail.refreshReferenceObject("subAccount");
-				SubAccount subAccount = subAccountGlobalDetail.getSubAccount();
+
+		for (SubAccountGlobalDetail subAccountGlobalDetail : subAccountGlobalDetails) {
+			subAccountGlobalDetail.refreshReferenceObject("subAccount");
+			SubAccount subAccount = subAccountGlobalDetail.getSubAccount();
+			
+			// update active flag
```

Comment:
These types of comments don't really seem to add anything as the code is pretty self-explanatory. My preference would be for them to be removed (if you need to make code changes from this review; if not, they can stay and this note can be for future reference).